### PR TITLE
tools/player: update progress bar UI

### PR DESCRIPTION
**Pull request by mbouron-gpsw**

### CODE REVIEW

File: ngl-tools/player.c [hunk 1]
```diff
@@ -329,15 +329,15 @@ static struct ngl_node *add_progress_bar(struct ngl_node *scene)
...
-    static const float text_corner[3] = {1.0 - 2.0 * 1/3., -1.0, 0.0};
-    static const float text_width[3]  = {2.0 * 1/3.,  0.0, 0.0};
-    static const float text_height[3] = {0.0,  2.0 * 0.05, 0.0};
-    static const float text_bg[4]     = {1.0, 1.0, 1.0, 0.0};
-    static const float text_fg[4]     = {0.0, 0.0, 0.0, 0.0};
+    static const float text_corner[3] = {-1.0, -1.0, 0.0};
+    static const float text_width[3]  = { 2.0,  0.0, 0.0};
+    static const float text_height[3] = {0.0, 0.1, 0.0};
+    static const float text_bg[4]     = {0.0, 0.0, 0.0, 0.0};
+    static const float text_fg[4]     = {1.0, 1.0, 1.0, 1.0};
```
Comment (cboesch-gpsw) [inline ngl-tools/player.c:341]: If you want to be consistent with what happens on `update_pgbar()`, you want to set `0.8` on the `text_bg` alpha, or `0.0` on the alpha of both `text_bg` and `text_fg`.

File: ngl-tools/player.c [hunk 2]
```diff
@@ -329,15 +329,15 @@ static struct ngl_node *add_progress_bar(struct ngl_node *scene)
...
+    static const float bar_height[3] = { 0.0,  2.0 * 0.01, 0.0}; // 1% of the height
 
-    static const float text_corner[3] = {1.0 - 2.0 * 1/3., -1.0, 0.0};
-    static const float text_width[3]  = {2.0 * 1/3.,  0.0, 0.0};
-    static const float text_height[3] = {0.0,  2.0 * 0.05, 0.0};
-    static const float text_bg[4]     = {1.0, 1.0, 1.0, 0.0};
-    static const float text_fg[4]     = {0.0, 0.0, 0.0, 0.0};
+    static const float text_corner[3] = {-1.0, -1.0, 0.0};
+    static const float text_width[3]  = { 2.0,  0.0, 0.0};
+    static const float text_height[3] = {0.0, 0.1, 0.0};
```
Comment (cboesch-gpsw) [inline ngl-tools/player.c:339]: Would you mind keeping `2.0 * 0.05` unchanged (it's equal to `0.1`), so we can see more obviously the ratio 5% for the text and 1% for the height?

**Review by cboesch-gpsw — COMMENTED** [review]
I like this a lot. Thanks.

File: ngl-tools/player.c (hunk 2 above)
Comment (mbouron-gpsw) [reply ngl-tools/player.c]: Done.

File: ngl-tools/player.c (hunk 1 above)
Comment (mbouron-gpsw) [reply ngl-tools/player.c]: Done.

**Review by cboesch-gpsw — APPROVED** [review]

**Pull request merged**